oneMoreSeive.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+function seiveBlocks(n) {
+  let primes = new Array();
+  let nPrimes = new Array(n + 1).fill(false);
```

Comment:
```suggestion
  let nPrimes = new Array(Math.floor(Math.sqrt(n)) + 1).fill(false);
```

Reply:
I use nPrimes later to cross out complex numbers.

Reply:
I know but the point of the algorithm is to not store n-length array otherwise it would be possible to just use the standard Sieve of Eratosthenes